src/main/java/cl/transbank/webpay/webpayplus/WebpayPlus.java

```diff
@@ -354,7 +356,7 @@ public static WebpayPlusMallTransactionStatusResponse status(String token, Optio
     }
 
     public static class MallDeferredTransaction {
-        @Getter(AccessLevel.PRIVATE) private static Options options = new Options();
+        @Getter(AccessLevel.PRIVATE) private static WebpayOptions options = new WebpayOptions();
```

Comment:
Use the instance of `Options` instead of `WebpayOptions` in order to be consistent with the other classes

Reply:
done